README.md

```diff
@@ -4,4 +4,6 @@
 
 If you want to contribute, please take a look at the [wiki](https://github.com/libgdx/libgdx.github.io/wiki) of this repo. For bigger changes, it is recommended to join our [Discord server](https://libgdx.com/community/discord/) and talk with the contributors about it. If you want to edit one of the wiki pages, those are located in the [wiki/](https://github.com/libgdx/libgdx.github.io/tree/dev/wiki) folder. Please also take a look at our [wiki style guide](https://libgdx.com/wiki/misc/wiki-style-guide).
 
+You can run the website locally using docker : `docker compose up jekyll`. In case of dependency changes (Gemfile or Gemfile.lock changes), you need to rebuild it before : `docker build .`.
```

Comment:
IMO it would be better to just refer to the wiki here (`To deploy the website locally, check out the instructions [here](https://github.com/libgdx/libgdx.github.io/wiki/Deploying-locally)` and then update the instructions there.

Reply:
OK i removed it, we'll have to update the wiki once mergd then.